src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/VSPackage.resx

```diff
@@ -138,4 +138,7 @@
   <data name="7" xml:space="preserve">
     <value>Loaded Project File Editor</value>
   </data>
+  <data name="8" xml:space="preserve">
+    <value>Unmigrated .NET Core Project Files (*.xproj);*.xproj</value>
```

Comment:
@srivatsn is there a different string we want to use for this? I took a stab, but I don't know if we want something more concise.